tests/playbooks/tests_ipv6_dns_search.yml

```diff
@@ -0,0 +1,93 @@
+---
+- hosts: all
+  tasks:
+    - import_role:
+        name: linux-system-roles.network
+      vars:
+        network_connections:
+          - name: br-example
+            type: bridge
+            ip:
+              dhcp4: false
+              auto6: false
+              dns: 8.8.8.8
+              dns_search: google.com
+              address:
+                - 10.2.2.77/26
+            state: up
+    - block:
+        - name: Verify nmcli connection DNS search entry for IPv4
+          shell: |
+            set -euxo pipefail
+            nmcli -f ipv4.dns-search connection show br-example
```

Comment:
Since these are technically integration tests and not functional tests, it might be nicer to check if /etc/resolv.conf contains "search example.com" or similar. I am not sure if this is feasible on systems that use systemd-resolved. Maybe resolvectl can be used, then. @ffmancera what do you think about this?

Reply:
Yes I agree. Ideally ´/etc/resolv.conf´ should show the values configured. Thanks and sorry for the delay.